vite.config.js

```diff
@@ -0,0 +1,13 @@
+import * as path from 'node:path'
+import { defineConfig } from 'vite'
+import vue from '@vitejs/plugin-vue'
+
+module.exports = defineConfig({
+  plugins: [vue()],
+  build: {
+    entry: path.resolve(__dirname, 'demo/index.js'),
+    outDir: path.resolve(__dirname, 'demo/dist'),
+    emptyOutDir: true,
+    minify: false,
+  },
```

Comment:
I think it can be simplified to the following, don't you?

```suggestion
  root: 'demo',
  plugins: [vue()],
  build: {
    emptyOutDir: true,
    minify: false,
  },
```